.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Just adding it here so folks can use IntelliJ with Rust plugin

Reply:
I also use intellij :muscle:, but have this ignored globally.

Doesn't hurt to include it here too tho.